test/JDBC/expected/sys-host_name-vu-prepare.out

```diff
@@ -0,0 +1,12 @@
+CREATE VIEW host_name_view AS SELECT DISTINCT hostname FROM sys.sysprocesses WHERE spid = @@SPID
```

Comment:
we should add tests for dm_exec_session and connection as well

Reply:
Added tests for the same.